cmd/internal/rssshrinker/rssshrinker_linux.go

```diff
@@ -0,0 +1,109 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Could we update the CODEOWNERS list so the right team own this.

Reply:
@hush-hush I realized too late that this PR is based on some old changes. if the codeowners is still out of date in `main`, you may be in a better position to update it than me. I don't know who the new owner is supposed to be ;)